src/metabase/metabot/command.clj

```diff
@@ -204,9 +196,9 @@
 
 (def ^:private kanye-quotes
   (delay
-    (log/debug (trs "Loading Kanye quotes..."))
-    (when-let [url (io/resource "kanye-quotes.edn")]
-      (edn/read-string (slurp url)))))
+   (log/debug (trs "Loading Kanye quotes..."))
```

Comment:
[The body of `delay` forms should be indented by 2 spaces](https://guide.clojure.style/#body-indentation), please revert this indentation change